smalltalksrc/VMMaker/DruidJIT.class.st

```diff
@@ -49,13 +49,31 @@ DruidJIT class >> initialize [
 	NumStoreTrampolines := 5
 ]
 
+{ #category : #'class initialization' }
+DruidJIT class >> initializeBytecodeTableForSistaV1 [
+
+	<generated>
+	numPushNilsFunction := #sistaV1:Num:Push:Nils:.
+	pushNilSizeFunction := #sistaV1PushNilSize:numInitialNils:.
+	BytecodeSetHasDirectedSuperSend := true.
+	BytecodeSetHasExtensions := true.
+	FirstSpecialSelector := 96.
+	NumSpecialSelectors := 32.
+	self generatorTableFrom: { 
+		#(1   0 223 unknownBytecode).
+		#(2 224 247 unknownBytecode).
+		#(3 248 255 unknownBytecode) }
```

Comment:
You wrote this, right? Because I'm working on this table generation

Reply:
yes :) i did it to test